### ISSUE TITLE
Dev Version (3/25/2018) - Missing value "gpu_threads_cong"

### ISSUE THREAD

**Issue by CircusDad** [fireice-uk/xmr-stak#1199]
I compiled the Dev version on 3/25/2018 on a windows machine.  This is the same computer on which I earlier compiled the 3/22/2018 Dev version without issue.  I then moved the executable to the miner (the same miner on which I properly ran the 3/22/2018 dev version).  I deleted cpu.txt, amd.txt and config.txt so they would be recreated.

Ran executable and got this result:

![invalid config file](https://user-images.githubusercontent.com/33241250/37878031-ff012e72-3030-11e8-8d2c-2b41005c8c66.png)

I then ran xmr-stak again but this time using the --noNVIDIA and --noAMD tags and it errored with a cpu related error I did not save (sorry).   I then ran the same command again and it worked (hashing with CPU only).

I then tried xmr-stak --noAMD    (I don't have an NVIDIA in this machine but wanted to ensure it was not related).  This time it also started and hashed with CPU only.

I then tried xmr-stak --noNVIDIA and the program does not run.  It flashes a black screen and goes away.

I then tried xmr-stak --version-long and it flashes a black screen and goes away.

**Comment by psychocrypt** [comment]
do you updated also the dll's? Please post the command you used to start
the miner and all output, incl. the configuration questions. (please change
your address in the log)

**Comment by CircusDad** [comment]
It could be the dll's!   I updated the dll's when I built and moved to my 3/22 dev build but this time I only copied over the new exe and reused the 3/22 dll's (assumed they were the same since xmr-dep did not change again).   Sorry.   I will try moving over the new dll set (and post your requested info) but it won't be till much later tonight as I had to head out.   Unless others jump on to this thread and affirm the same issue, I would assume it was dll related user error.
